frontend/src/pages/Mesh/Mesh.tsx

```diff
@@ -247,15 +248,17 @@ const TopologyContent: React.FC<{
 
       function addNode(data: NodeData): NodeModel {
         data.onHover = onHover;
+        console.log(`infraType=${data.infraType}`);
```

Comment:
`console.log` is not recommended. If this message is for debugging purposes, please change it to `console.debug`

Reply:
That shouldn't be there at all, I'll remove it.  It's already gone in my next PR...